wolfssl/wolfcrypt/integer.h

```diff
@@ -311,6 +311,7 @@ MP_API void mp_free (mp_int * a);
 MP_API void mp_forcezero(mp_int * a);
 MP_API int  mp_unsigned_bin_size(const mp_int * a);
 MP_API int  mp_read_unsigned_bin (mp_int * a, const unsigned char *b, int c);
+MP_API int  mp_read_unsigned_bin_ct (mp_int * a, const unsigned char *b, int c);
```

Comment:
integer.c implementation isn't constant time.
Map it to the normal funciton.

Reply:
fixed